webpack.config.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-env node */
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const webpack = require('webpack');
+const path = require('path');
+
+const staticFilesDir = path.resolve('contentcuration', 'contentcuration', 'static');
+const staticJsDir = path.resolve(staticFilesDir, 'js');
+const staticLessDir = path.resolve(staticFilesDir, 'less');
+
+const bundleEntryDir = path.resolve(staticJsDir, 'bundle_modules');
+const bundleOutputDir = path.resolve(staticJsDir,'bundles');
+
+const jsLoaders = [
+  {
+    loader: 'babel-loader',
+    options: {
+      // might be able to limit browsers for smaller bundles
+      presets: ['env'],
+    },
+  }
+];
+
+// NOTE: Lots of things are handled by webpack4. NODE_ENV, uglify, source-maps
```

Comment:
But where are you setting `NODE_ENV`?

Reply:
Or, rather, where do we toggle the `mode` between `development` and `production`?

Reply:
I put it in the dockerfile, as it was the only place the `--production` flag was being used with `build.js`

Reply:
I think webpack 4 actually sets the `NODE_ENV` now, too?
> Defines the `process.env.NODE_ENV` constant to a compile-time-constant value. This allows to remove development only code from code.
https://medium.com/webpack/webpack-4-mode-and-optimization-5423a6bc597a

Reply:
Yeah, that's why I changed my question to the second one :)